Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -90,72 +108,54 @@ Observe the overall shape of the average profiles. Does the effect appear to inc
 
 ```{r, cache = TRUE, fig.width = 10, fig.height = 6, warning = FALSE, message= FALSE}
 data_to_plot <- list()
-data_to_plot[[1]] <- my.data[my.data$CMT==2,]
-data_to_plot[[2]] <- my.data[my.data$CMT==3,]
+data_to_plot[[1]] <- pk_data
```

Comment:
Andy removed the for loop for the PKPD plots in the PKPD_binary page as they weren't really necessary, so we can do that here too. Having one plot after the other is not a big deal and less confusing than a loop.